providers/qedr/qelr_verbs.c

```diff
@@ -333,6 +508,11 @@ static void qelr_chain_free_rq(struct qelr_qp *qp)
 	qelr_chain_free(&qp->rq.chain);
 }
 
+static inline bool qelr_qp_has_srq(struct qelr_qp *qp)
+{
+	return !!qp->srq;
```

Comment:
!! obfuscation isn't needed if casting to bool

Reply:
will fix.